src/DOMTestCase.php

```diff
@@ -252,8 +252,8 @@ private static function filterCrawler(Crawler $crawler, $content)
                 return $node->text() === '';
             }
 
-            if (preg_match('/^regexp\s*:\s*(.*)/i', $content, $matches)) {
-                return (bool) preg_match($matches[1], $node->text());
+            if (preg_match('/^regexp\s:*\s*(?P<matchName>.*)/i', $content, $matches)) {
```

Comment:
matchName was just an example, you can give it any name, in this case `expression` is probably a good idea.